src/tools/gophish_complete.py

```diff
@@ -167,14 +167,15 @@ def main() -> None:
         )
     except ValueError:
         logging.critical(
-            f'"{log_level}"is not a valid logging level. Possible values are debug, info, warning, and error.'
+            '"%s"is not a valid logging level. Possible values are debug, info, warning, and error.',
```

Comment:
I _think_ you need an extra space here.
```suggestion
            '"%s" is not a valid logging level. Possible values are debug, info, warning, and error.',
```

Reply:
See 825962b